pkg/api/customization/cluster/formatter.go

```diff
@@ -145,6 +148,12 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 		}
 	}
 
+	nodes, err := f.nodeLister.List(resource.ID, labels.Everything())
+	if err != nil {
+		logrus.Warnf("error getting node list for cluster %s: %s", resource.ID, err)
+	} else {
+		resource.Values["nodeCount"] = len(nodes)
```

Comment:
We will be showing the nodeCount, irrespective of the cluster state; Also we list all nodes irrespective of the node status. Just confirming we want is this way? Also,  can we add an integration test to check the nodeCount of a cluster created?

Reply:
That is the current behavior when using the CLI